### DISCUSSION TITLE
Remove LandUseDataSet and LandCoverDataSet?

### DISCUSSION THREAD

**Discussion by JohannaOtt** [INSPIRE-MIF/helpdesk#120] [Q&A]
There used to be a [discussion in the INSPIRE Forum](https://wayback.archive-it.org/12090/20210119163447/https://inspire.ec.europa.eu/forum/discussion/view/245981/discussion-about-amendmends-for-tg-land-cover-and-land-use) to remove the LandUseDataSet and the LandCoverDataSet from the respective schemas because they cause multiple issues.
More detailes description can be found in the old thread.
Lena Hallin-Pihlatie mentioned back then that there should be a list with changes compiled by the Community Forum facilitators and that this change request would be added there.
Does anyone know if this was discussed/addressed anywhere?

**Comment by sMorrone** [comment]
Dear Johanna,
IR change proposals from the INSPIRE Community Forum were presented to the 8th MIG meeting. Approved changes were included in the IR draft amendment text and are now in the Comitology procedure phase. 
However, proposals for LU and LC did not deal with deletion of LandUseDataSet and the LandCoverDataSet from the respective schemas. Conversely,  for the LU the proposal was to change the data type of the _extent_ attribute (from current _GM_MultiSurface_ to _EX_Extent_), whilst for LC it was proposed to simplify the "aggregation" relashionship between LandCoverDataSet and LandCoverUnit.  In this latter case, however, no more details were provided on how to implement the simplification - e,g, turning aggregation into simple association as proposed[ in this discussion](https://wayback.archive-it.org/12090/20161216094638/https://themes.jrc.ec.europa.eu/discussion/view/48703/the-association-role-%E2%80%99member%E2%80%99-in-land-cover-and-land-use) from Thematic Clusters.

**Comment by JohannaOtt** [comment]
@sMorrone Thank you for the feedback.
I would propose to address that issue again and potentially create a change request on that issue.
Any opinions from the community on deleting the two as discussed in the forum back in the days?
Any arguments for keeping them?

**Comment by KathiSchleidt** [comment]
@JohannaOtt if the LC and LU types have been harmonized (both now utilizing EX_Extent to provide the extent, as already implemented in LC), and the aggregation between LCDataset and LCUnit taken down to an association, both models should be implementable.

As for removing the dataset types, I'd worry that this would cause real issues at IR level. Also, as long as the MD level insists on a dataset concept, kinda nice to have this reflected in the model (e.g., if you only have a reference to one LCUnit, this way you can easily find the siblings)

What arguments do you have for removing entirely?

**Reply by JohannaOtt** [reply]
> if the LC and LU types have been harmonized (both now utilizing EX_Extent to provide the extent, as already implemented in LC)

@KathiSchleidt Do you know if there are any plans to do so?

**Comment by JohannaOtt** [comment]
Basically still the same ones as in the old thread:
* there is not a lot of relevant information in the DataSets and it could also be provided in the metadata
* there are issues with these objects when creating WFS because they have so many references that the objects are huge
* (issues with the extent - seems like they were addressed in the proposal Stefania mentioned)
